File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.py
def solution(my_string):
    # 1. 문자열을 소문자로 변환
    lowstr = my_string.lower()

    # 2. 소문자로 변환된 문자열을 정렬 (알파벳 순서대로 정렬됨)
    sort_str = sorted(lowstr)

    # 3. 정렬된 리스트를 문자열로 변환하여 반환
    answer = "".join(sort_str)

    return answer

# ✅ 예제 테스트 실행
print(solution("Bcad"))
# 결과: "abcd" (소문자로 변환 후 정렬)

print(solution("heLLo"))
# 결과: "ehllo" (소문자로 변환 후 정렬)

print(solution("Python"))
# 결과: "hnopty" (소문자로 변환 후 정렬)

File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.java
import java.util.Arrays;

public class Solution {

    public static String solution(String my_string) {
        // 1. 문자열을 소문자로 변환
        String lower = my_string.toLowerCase();

        // 2. 문자 배열로 변환 후 정렬
        char[] chars = lower.toCharArray();
        Arrays.sort(chars);  // 알파벳 순서대로 정렬

        // 3. 정렬된 문자 배열을 문자열로 변환
        return new String(chars);
    }

    // ✅ 테스트 실행
    public static void main(String[] args) {
        System.out.println(solution("Bcad"));    // abcd
        System.out.println(solution("heLLo"));   // ehllo
        System.out.println(solution("Python"));  // hnopty
    }
}
